src/wallet/rpcwallet.cpp

```diff
@@ -4762,6 +4895,9 @@ UniValue z_listreceivedbyaddress(const UniValue& params, bool fHelp)
         obj.pushKV("amount", ValueFromAmount(CAmount(entry.plaintext.value())));
         std::string data(entry.plaintext.memo().begin(), entry.plaintext.memo().end());
         obj.pushKV("memo", HexStr(data));
+        // (txid, jsindex, jsoutindex) is needed to globally identify a note
+        obj.pushKV("jsindex", entry.jsop.js);
+        obj.pushKV("jsoutindex", entry.jsop.n);
```

Comment:
Please, add the description of the added fields the the function dcoumentation, as done in `UniValue z_listunspent(const UniValue& params, bool fHelp)`

Reply:
Fixed in https://github.com/HorizenOfficial/zen/pull/446/commits/5378dbf723b880a8bab27983ea0947356512973d